homework/homework.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"fmt"
+	"gocourse05/core/camera"
+	"math/rand/v2"
+)
+
+type ID interface {
+	GetId() int
+}
+
+type Metadata interface {
+	GetMetaData() string
+}
+
+func main() {
+	camerasList := buildCamerasList()
+
+	data := map[int]string{}
+	photoID := 1
+	for _, zooCamera := range camerasList {
+		photoData := ""
+		// Check if camera is instance of ID interface
+		if IDCamera, ok := zooCamera.(ID); ok {
```

Comment:
локальні змінні завжди з маленької:
```suggestion
		if cameraID, ok := zooCamera.(ID); ok {
```